src/expr/node_manager.h

```diff
@@ -34,6 +34,8 @@
 
 namespace cvc5 {
 
+using Record = std::vector<std::pair<std::string, TypeNode>>;
```

Comment:
can you document how this will be used?

Reply:
This will be moved from record.h in the other PR, I'll remove from this PR.